workflows-scripts/EBSCONet/EBSCONET.jmx

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.4.1">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="EBSCONET" enabled="true">
```

Comment:
Please change the test name and anywhere else that refers to "EBSCONET" to "EBSCONet-order-renewal" (or some variant of the casing that adheres to the PTF established naming convention). 

```suggestion
    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="EBSCONet Order Renewal" enabled="true">
```